macstadium-shared-2/main.tf

```diff
@@ -404,6 +404,16 @@ module "vsphere_janitor_custom_5" {
   index = "${var.index}"
 }
 
+module "vsphere_monitor" {
+  source = "../modules/vsphere_monitor"
+  host_id = "${module.macstadium_infrastructure.util_uuid}"
+  ssh_host = "${module.macstadium_infrastructure.util_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.vsphere_monitor_version}"
```

Comment:
Same for this -- waiting on https://github.com/travis-ci/vsphere-monitor/issues/7 to be solved first.